local-cli/util/Config.js

```diff
@@ -150,14 +150,15 @@ function getProjectPath() {
   return path.resolve(__dirname, '../..');
 }
 
-const resolveSymlink = (roots) =>
-  roots.concat(
-    findSymlinksPaths(
-      path.join(getProjectPath(), 'node_modules'),
-      roots
-    )
+const resolveSymlinksForRoots = roots =>
```

Comment:
Adds support for resolving symlinks in multiple roots, rather than just the project path